src/server/editorServices.ts

```diff
@@ -754,6 +756,103 @@ namespace ts.server {
             return false;
         }
 
+        acquireTypingForJs(project: Project) {
+            if (!this.host.getTsd) {
+                return;
+            }
+            const tsd = this.host.getTsd();
+            if (project === undefined || tsd === undefined) {

```

Comment:
Do you mean
```ts
let tsd: any;
if (!this.host.getTsd || !this.host.getTsd() || project === undefined) {
    return;
}
tsd = this.host.getTsd();
```
Though I still need to call the `getTsd` twice to assign the result to a local variable `tsd`.